packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -543,12 +543,9 @@ class CodeViewEmptyState extends StatelessWidget {
   const CodeViewEmptyState({
     super.key,
     required this.widget,
-    required this.context,
   });
 
   final CodeView widget;
-  final BuildContext context;
-
```

Comment:
Removal was done automatically, so some newlines might be absent. Will restore this one while applying other review comments.